### PR TITLE
K8SPSMDB-341 Minor tests fixes

### PR DESCRIPTION
**Pull request by cap1984**
[K8SPSMDB-341](https://jira.percona.com/browse/K8SPSMDB-341)

- Monitoring compare file fix (PMM server app version update)
- Check additional sharding parameter (https://github.com/percona/percona-server-mongodb-operator/pull/497#discussion_r539967937)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| one-pod | passed |
| storage | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| demand-backup | failed |
| self-healing | passed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| scheduled-backup | failed |

File: e2e-tests/rs-shard-migration/run [hunk 1]
```diff
@@ -48,7 +49,8 @@ function main() {
 
     simple_data_check "${cluster}" "${CLUSTER_SIZE}" 1 "-mongos"
 
-    if [[ -z "$(get_lastCommitedOpTime ${cluster} ${namespace})" ]]; then
+    if [[ -z "$(get_shard_parameter ${cluster} ${namespace} lastCommitedOpTime)" ]] \
```
Comment (tplavcic) [inline e2e-tests/rs-shard-migration/run:52]: If I see correctly lastCommitedOpTime is not available always for 3.6 so it is expected to fail there: https://jira.mongodb.org/browse/SERVER-33018
This was fixed in 3.7 and higher versions.
Also I don't understand the logic to check this on rs0 for checking sharding.

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: e2e-tests/rs-shard-migration/run (hunk 1 above)
Comment (cap1984) [reply e2e-tests/rs-shard-migration/run:52]: actually it's executed on ``-rs0-0``

Comment (cap1984) [reply e2e-tests/rs-shard-migration/run:52]: updated the comment to the actual reason

**Review by tplavcic — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| operator-self-healing | passed |
| init-deploy | passed |
| one-pod | passed |
| limits | passed |
| monitoring-2-0 | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| upgrade-consistency | passed |
| demand-backup | failed |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| demand-backup | failed |
| monitoring-2-0 | passed |
| self-healing | passed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| scheduled-backup | failed |

**Pull request merged**